docs/authentication/social-connections/spotify.mdx

```diff
@@ -1,65 +1,72 @@
 ---
 title: Add Spotify as a social connection
-description: Learn how to set up a social connection with Spotify in your Clerk application.
+description: Learn how to allow users to sign in and sign up for your Clerk application with their Spotify account.
```

Comment:
small nit: i like "sign up" to be before "sign in" and i try to keep that consistent thru out the docs!

```suggestion
description: Learn how to allow users to sign up and sign in to your Clerk application with their Spotify account.
```